Makefile

```diff
@@ -16,9 +16,13 @@ API_JAR=jar/async-profiler.jar
 CONVERTER_JAR=jar/jfr-converter.jar
 TEST_JAR=test.jar
 
-CC=$(CROSS_COMPILE)gcc
-CXX=$(CROSS_COMPILE)g++
-STRIP=$(CROSS_COMPILE)strip
+CC ?= gcc
+CXX ?= g++
+STRIP ?= strip
+
+CC := $(CROSS_COMPILE)$(CC)
+CXX := $(CROSS_COMPILE)$(CXX)
+STRIP := $(CROSS_COMPILE)$(STRIP)
```

Comment:
This is not a compatible change.
Now, `make CROSS_COMPILE=aarch64-linux-gnu-` will try to execute non-existent `aarch64-linux-gnu-cc`, whereas the actual binary is ``aarch64-linux-gnu-gcc`.